src/layouts/home-assistant-main.js

```diff
@@ -176,11 +177,7 @@ class HomeAssistantMain extends NavigateMixin(EventsMixin(PolymerElement)) {
   }
 
   computeRTL(hass) {
```

Comment:
`_computeRTL` This shows its a private function and differentiates it from the `computeRTL` function